### PR TITLE
Adding Solidity Summit Recap Post and Imgs to the Blog

### PR DESCRIPTION
**Pull request by franzihei**
closes https://github.com/ethereum/solidity-blog/issues/61

### CODE REVIEW

File: _posts/2020-06-08-solidity-summit-recap.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+published: true
+title: 'Wrapping up the Virtual Solidity Summit 2020'
+date: '2020-06-08'
```
Comment (chriseth) [inline _posts/2020-06-08-solidity-summit-recap.md:5]: Needs to be updated.

File: _posts/2020-06-08-solidity-summit-recap.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+Initially, the Solidity Summit was planned to be a small, language design focused in-person gathering in Berlin with a expected participant count of 100-150 people.
+
+Since the global circumstances forced us to [transform it into an online event](https://solidity.ethereum.org/2020/04/17/solidity-summit-2020-goes-interspace/), there was luckily no need for limits on participation required anymore. The event was free of charge and the livestreams were publicly accessible for anybody to join.
+
+**Solidity Summit 2020 by the numbers..**
+* Prior to the event, we received roughly 250 applications via the signup form.
+* During the event, the maximum combined participation of livestream and video conference room reached 150 concurrent participants, with 50-60 being the lowest concurrent participation rate.
+* Both livestreams for conference day 1 and 2 reached more than 1000 playbacks each.
+* The conference was single track event and featured 23 talks and 13 open discussion rounds in total.
```
Comment (chriseth) [inline _posts/2020-06-08-solidity-summit-recap.md:30]:
```suggestion
* The conference was a single track event and featured 23 talks and 13 open discussion rounds in total. 
```

File: _posts/2020-06-08-solidity-summit-recap.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+![Solidity Summit Agenda Day 2](/img/2020/06/AgendaDay2.png)
+*Solidity Summit Agenda - Day 2*
+
+## What did we learn? 💡
+
+Organizing an online conference for the first time was an exciting and challanging task. Here are a few learnings and takeways, which we would like to share:
+
+* **Finding the right tools is essential.** Which tools can best facilitate an open and interactive discussion online? How many tools are needed for a truly interactive expereience? If possible, can we support open source tooling without sacrificing good UX?
+* **Online conferencing can be quite exhausting.** Next time we'll do more breaks!
+* **Open discussions on language design questions are hard**. Good collaboration tools, preparation and moderation are important to facilitate the sessions and help structure them.
```
Comment (chriseth) [inline _posts/2020-06-08-solidity-summit-recap.md:47]:
```suggestion
* **Open discussions on language design questions are hard.** Good collaboration tools, preparation and moderation are important to facilitate the sessions and help structure them.
```

File: _posts/2020-06-08-solidity-summit-recap.md (hunk 1 above)
Comment (axic) [reply _posts/2020-06-08-solidity-summit-recap.md]: If date is changed the file should also be renamed.

**Pull request merged**